src/components/GameDropzone.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useGamesStore } from '@/stores/games'
+
+const gamesStore = useGamesStore()
+const fileInput = ref<HTMLInputElement | null>(null)
```

Comment:
Is this used?

Reply:
https://github.com/ZetaTwo/aoe2replaypacker/pull/23/files/c1e17aa47906ed0992cce96a8740fdce367aa11f#diff-2994847ec2277cb8e7cd2d075a3c9fe83807945549d4e8f724ed223a7aab7a7dR44

Reply:
Yes, but why is that ref attribute needed?